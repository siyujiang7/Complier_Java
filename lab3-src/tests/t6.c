char*x,y;long main(){}long d,y,eee;double notmain(){}
