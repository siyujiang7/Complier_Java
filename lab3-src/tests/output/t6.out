char* x;
char* y;
long main()
{
}
long d;
long y;
long eee;
double notmain()
{
}
